src/features/workspaceSymbolProvider.ts

```diff
@@ -23,10 +23,15 @@ export default class OmnisharpWorkspaceSymbolProvider extends AbstractSupport im
         let options = this.optionProvider.GetLatestOptions();
         let minFilterLength = options.minFindSymbolsFilterLength > 0 ? options.minFindSymbolsFilterLength : undefined;

```

Comment:
>minFindSymbolsFilterLength [](start = 38, length = 26)

Sounds like you could remove the setting as well.

Reply:
the setting makes sense, it's just we evaluate it on the client rather than server

Reply:
you are right - sorry for the confusion.